global.json

```diff
@@ -0,0 +1,5 @@
+{
+    "sdk": {
+        "allowPrerelease": false
```

Comment:
Curious about this. Was there a feature we are trying to access that is in preview?

Reply:
I use Visual Studio 2019 Preview, and have preview SDKs installed on my machine - this just hides a compile-time error for me - doesn't hurt or hinder anyone else but I will remove.